test/functional/TestUtilities/src/org/openj9/test/util/VersionCheck.java

```diff
@@ -33,6 +33,9 @@
 	static final Object versionInstance;
 	static final Method majorMethod;
 	
+	public static final int JAVA8 = 8;
+	public static final int JAVA14 = 14;
```

Comment:
I would prefer not to add these constants as it becomes one more thing to remember to update each release

Reply:
So is it preferred to have a check like `VersionCheck.major() >= 14` instead?

Reply:
yes

Reply:
Removed the changed in `VersionCheck` and updated the usages of `major()`.
Please review.